Transformer/PythonCheckpointReader.swift

```diff
@@ -0,0 +1,116 @@
+import TensorFlow
+
+struct Config {
+    let vocabSize: Int
+    let contextSize: Int
+    let embeddingSize: Int
+    let headCount: Int
+    let layerCount: Int
+}
+
+extension Config {
+    init(dict: [String: Int]) {
+        vocabSize = dict["n_vocab"]!
+        contextSize = dict["n_ctx"]!
+        embeddingSize = dict["n_embd"]!
+        headCount = dict["n_head"]!
+        layerCount = dict["n_layer"]!
+    }
+}
+
+let config = Config(dict: [
+    "n_vocab": 50257,
+    "n_ctx": 1024,
+    "n_embd": 768,
+    "n_head": 12,
+    "n_layer": 12])
+
+func readTensor<Scalar: TensorFlowScalar>(
+    from path: String, name: String, scalarType: Scalar.Type
+) -> Tensor<Scalar> {
+    // TODO(jekbradbury): support variadic dtype attrs in RawOpsGenerated
+    return Tensor<Scalar>(handle: #tfop(
```

Comment:
Generic parameter is redundant.

```suggestion
    return Tensor(handle: #tfop(
```